packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/mira/MetaRegistryProxy.java

```diff
@@ -0,0 +1,12 @@
+package software.uncharted.terarium.hmiserver.proxies.mira;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+
+@RegisterRestClient(configKey = "mira-metaregistry")
+@Produces(MediaType.APPLICATION_JSON)
+public interface MetaRegistryProxy {
```

Comment:
What is this empty interface for?

Reply:
It exist, I wanted to use it, but that was simple to send the URL to the front end instead of resolving in the back-end.